### ISSUE TITLE
steps

### ISSUE THREAD

**Issue by kareem1925** [ahmed-tea/tessdata_Arabic_Numbers#1]
can you please provide the steps you used?

**Comment by ahmed-tea** [comment]
@kareem1925 Steps for what?

**Comment by faisalshahbaz** [comment]
for training step.

**Comment by kareem1925** [comment]
@ahmed-tea training and preparing the data

**Comment by maliksyria** [comment]
yes Mr . Ahmad could you please provide us how to make such a file 
thanks in advance

**Comment by m0beenali** [comment]
@ahmed-tea please tell how you changed AEN to AWN while training? I'have trained file with {٠١٢٣٤٥٦٧٨٩/} i need the / also as i am reading dates in arabic. Please help @ahmed-tea

**Comment by MahmoudMabrok** [comment]
@m0beenali  
@maliksyria 
i think this may help 

https://github.com/tesseract-ocr/tesseract/issues/1193#issuecomment-414124398
